README.md

```diff
@@ -1,2 +1,11 @@
-# coding-standards
-ARC's recommended coding standards
+# Coding standards
+
+UCL Advanced Research Computing Centre's recommended coding standards for research software projects
+
+  * [C++](cpp)
+  * [Fortran](fortran)
+  * [Java](java)
+  * [Julia](julia)
+  * [MATLAB](matlab)
+  * [Python](python)
+  * [R](r)
```

Comment:
```suggestion
  * [Python](python)
  * [R](r)
```

I might hide everything at the start and unhide as the `lang/README.md`'s become populated.

Reply:
Agreed better to have only links to READMEs that are actually populated. I'll also remove the placeholder files, mainly added these to test how things displayed on GitHub pages.